core/src/main/scala/org/apache/spark/internal/config/Status.scala

```diff
@@ -70,4 +70,11 @@ private[spark] object Status {
       .version("3.0.0")
       .booleanConf
       .createWithDefault(false)
+
+  val LIVE_UI_LOCAL_STORE_DIR = ConfigBuilder("spark.ui.store.path")
+    .doc("Local directory where to cache application information for live UI. By default this is " +
+      "not set, meaning all application information will be kept in memory.")
```

Comment:
this needs to be documented in the .md docs unless you have a followup issue to document all at once?

Reply:
also how does this or will this interact with  spark.history.store.path?  do we want to explicitly put live in the name of the config to make it clear if they are totally separate?

Reply:
@tgravescs Thanks, I have added the docs in `docs/configuration.md`

Reply:
> how does this or will this interact with spark.history.store.path? do we want to explicitly put live in the name of the config to make it clear if they are totally separate?

If the RocksDB instance is gracefully closed, SHS can set the backend as RocksDB and start with the instance from the live UI.